src/codemodder/codemods/base_codemod.py

```diff
@@ -7,7 +7,7 @@
 from dataclasses import dataclass, field
 from enum import Enum
 from functools import cached_property
-from importlib.abc import Traversable
+from importlib.resources.abc import Traversable
```

Comment:
Fix deprecation warning

Reply:
Unfortunately it looks like we need to wait until we drop 3.10 support for this one (but we can do that pretty much any time we want).